test/array.every.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const tap = require('tap');
+const metasync = require('..');
+
+function identity(x, callback) {
```

Comment:
Because it's used in tests but now `metarhia/common` doesn't have it. I am going to create issue in `metarhia/common` about `identity` and other higher-order functions that I commonly use with Haskell. I think it will be useful to think advantages and disadvantages of them. Maybe some of them can be wonderfully adopted to Javascript like functor abstraction.